examples/aperture/input_aperture.in

```diff
@@ -40,7 +40,7 @@ collimator.aperture_y = 1.5e-3
 # Algorithms
 ###############################################################################
 algo.particle_shape = 2
-algo.space_charge = false
```

Comment:
I have one concern about this recent change.  With `algo.space_charge = false`, it is obvious to the user that space charge is not active.  Without this line, I think it is too easy to forget that space charge is off by default, _especially_ when a nonzero bunch charge is provided.  This is especially important in the examples.  If we keep three string input options "2D", "3D", and "false", then we could keep this line.

Reply:
I was wondering about that, good point.

Off was and is the default; I agree we can keep it explicit to avoid any confusion.